storage/storage-interface/src/cached_state_view.rs

```diff
@@ -89,6 +91,12 @@ pub struct CachedStateView {
     proof_fetcher: Arc<AsyncProofFetcher>,
 }
 
+impl Debug for CachedStateView {
```

Comment:
Do we really need this?

Reply:
Yes, unfortunately this is needed for Arc::try_unwrap function weirdly.